docs/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- `PromReporter` no longer reads directly from the db, and instead uses the txStore API.
```

Comment:
nit: change to: no longer directly reads txm related status from the db, and instead ...